### ISSUE TITLE
Logo in navbar has empty link

### ISSUE THREAD

**Issue by erikkroes** [electron/electronjs.org#4578]
There is a logo in the header that links to the home page. The logo is hidden for Assistive Technology like Screen Readers. There is no textual content inside the link. People who use Assistive Technology will not be able to understand where the link leads. This empty link is also not good for SEO.

A possible solution is a visually hidden text inside the link that describes the link purpose.

The corresponding WCAG criterion can be found here: https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html

===

This finding of this issue has been recorded. It is part of my weekly accessibility stream on Twitch: twitch.tv/erikKroes
A recording should be up soon at youtube: https://www.youtube.com/channel/UCwrpGO76k2HyUQspl-rG2eQ/videos

**Comment by tt-le** [comment]
@erickzhao Can I get assigned to this issue?

**Comment by erickzhao** [comment]
Hey @tt-le, for sure! If you need any help, feel free to join us at https://discord.gg/electron.
